src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 33px 17px 17px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  box-sizing: border-box;
+
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: $card-hover;
+    }
+
+    .card__btn-buy {
+      background-color: $white-color;
+      color: $blue-color;
+      border: solid 1px $blue-color;
+    }
+  }
+
+  &__image {
+    margin: 0 auto 40px;
+
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
```

Comment:
The width and height of the `.card__image` are hardcoded. It's better to make them more flexible by using relative units or by controlling the size with the parent container, to ensure better responsiveness.